allennlp/training/util.py

```diff
@@ -437,3 +437,22 @@ def description_from_metrics(metrics: Dict[str, float]) -> str:
     return ', '.join(["%s: %.4f" % (name, value)
                       for name, value in
                       metrics.items() if not name.startswith("_")]) + " ||"
+
```

Comment:
This shouldn't be a private function because you're using it in another file, just remove the preceeding underscore